converter.go

```diff
@@ -63,18 +63,9 @@ func resizeGeneric(in image.Image, out *image.NRGBA64, scale float64, coeffs []i
 					case xi >= maxX:
 						xi = maxX
 					}
+					// Forward alpha-premultiplication (if needed)
```

Comment:
This comment doesn't help much, as semantically something else is happening here.